kata-webhook/README.md

```diff
@@ -10,6 +10,7 @@ Kubernetes YAML files required to instantiate the admission
 controller.
 
 ```bash
+$ cp -a ../vendor .
```

Comment:
I wonder how expensive is this `cp` and if it could be replaced with a symbolic link (ln -s)

Reply:
Docker will **not** use files outside of the current build directory.
That's the reason we cannot do a `ln -s` ... or even do the copy inside the container.

Reply:
yeah - make sense, thanks